### PR TITLE
Add C# class source generation

### PR DESCRIPTION
**Pull request by ygorsilv1997**
**PR Title**
C# class generator to be used with dictionary hashtable

**PR Description**
To convert the hashtable in your class you can use the "ToModel" extension of the hashtable.

### CODE REVIEW

File: src/JJMasterData.Commons/Extensions/StringExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+using System.Globalization;
+using System.Text;
+using System.Linq;
+
+namespace JJMasterData.Commons.Extensions
+{
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Remove acentos da string
```
Comment (gumbarros) [inline src/JJMasterData.Commons/Extensions/StringExtensions.cs]: Maybe translate to English this comment?

Comment (ygorsilv1997) [reply src/JJMasterData.Commons/Extensions/StringExtensions.cs]: Adjusted

File: src/JJMasterData.Commons/Extensions/StringExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        /// </summary>
+        /// <param name="text"></param>
+        /// <returns></returns>
+        public static string RemoveAccents(this string text)
```
Comment (LucioPelinson) [inline src/JJMasterData.Commons/Extensions/StringExtensions.cs]: This method already exists in StringManager.NoAccents.
Please avoid Extensions methods to string

File: src/JJMasterData.Commons/Extensions/StringExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            var arrayText = text.Normalize(NormalizationForm.FormD).ToCharArray();
+            foreach (char letter in arrayText)
+            {
+                if (CharUnicodeInfo.GetUnicodeCategory(letter) != UnicodeCategory.NonSpacingMark)
+                    sbReturn.Append(letter);
+            }
+            return sbReturn.ToString();
+        }
+
+        public static string FirstCharToUpperComplete(this string input)
```
Comment (LucioPelinson) [inline src/JJMasterData.Commons/Extensions/StringExtensions.cs]: This method already exists in StringManager.FirstCharToUpper
Please remove this class

**Review by LucioPelinson — CHANGES_REQUESTED** [review]
Please remove class StringExtensions

File: src/JJMasterData.Commons/JJMasterData.Commons.csproj [hunk 4]
```diff
@@ -16,6 +16,9 @@
 		<LangVersion>10.0</LangVersion>
 		<Nullable>disable</Nullable>
 	</PropertyGroup>
+	<ItemGroup>
+	  <Compile Remove="Extensions\StringExtensions.cs" />
```
Comment (gumbarros) [inline src/JJMasterData.Commons/JJMasterData.Commons.csproj]: Please remove this <ItemGroup> from .csproj. The file does not exist.

**Review by gumbarros — APPROVED** [review]

**Pull request merged**